### PR TITLE
trawl: add page

### PR DESCRIPTION
**Pull request by Aracki**
I have added [trawl](https://github.com/robphoenix/trawl) - The package that does pretty printing network interface information to the console, much like `ifconfig/ipconfig/ip/ifdata`.

### CODE REVIEW

File: pages/common/trawl.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# trawl
+
+> Prints out network interface information to the console, much like ifconfig/ipconfig/ip/ifdata.
+
```
Comment (sbrl) [inline pages/common/trawl.md]: Could we add a line here for the homepage?

```markdown
> Homepage: <https://github.com/robphoenix/trawl>.

Comment (Aracki) [reply pages/common/trawl.md]: I didn't see that in other pages; but LGTM.

Comment (pxgamer) [reply pages/common/trawl.md]: It's relatively new, so most pages don't have them yet. :+1:

Comment (Aracki) [reply pages/common/trawl.md]: I agree with injecting links into the markdown!

**Pull request merged**